backend/coins/btc/handlers/handlers.go

```diff
@@ -189,14 +190,18 @@ func (handlers *Handlers) getTxInfoJSON(txInfo *accounts.TransactionData, detail
 		Unit:   amount.Unit,
 	}
 	var formattedTime *string
-	if txInfo.Timestamp != nil {
-		t := txInfo.Timestamp.Format(time.RFC3339)
+	var deductedAmount FormattedAmount
+	timestamp := txInfo.Timestamp
+	if timestamp == nil {
+		timestamp = txInfo.CreatedTimestamp
+	}
+	if timestamp != nil {
+		t := timestamp.Format(time.RFC3339)
 		formattedTime = &t
 		amountAtTime = handlers.formatAmountAtTimeAsJSON(txInfo.Amount, txInfo.Timestamp)
-	} else if txInfo.CreatedTimestamp != nil {
-		t := txInfo.CreatedTimestamp.Format(time.RFC3339)
-		formattedTime = &t
-		amountAtTime = handlers.formatAmountAtTimeAsJSON(txInfo.Amount, txInfo.CreatedTimestamp)
+		if txInfo.Fee != nil && txInfo.Type == accounts.TxTypeSend {
+			deductedAmount = handlers.formatAmountAtTimeAsJSON(coin.SumAmounts(txInfo.Amount, *txInfo.Fee), txInfo.Timestamp)
```

Comment:
```suggestion
			deductedAmount = handlers.formatAmountAtTimeAsJSON(coin.SumAmounts(txInfo.Amount, *txInfo.Fee), timestamp)
```
Here and also for `amountAtTime`